bot/search/elastic_search_manager.py

```diff
@@ -41,74 +41,98 @@ async def delete_all_indices(es: AsyncElasticsearch, logger: logging.Logger) ->
-            readable_output = f"Document ID: {response["hits"]["hits"][0]["_id"]}\n" \
-                              f"Episode Info: {document["episode_info"]}\n" \
-                              f"Video Path: {document["video_path"]}\n" \
-                              f"Segment Text: {document.get("text", "No text available")}\n" \
-                              f"Timestamp: {document.get("timestamp", "No timestamp available")}"
-            await log_system_message(logging.INFO, "Retrieved document:\n" + readable_output, logger)
+            readable_output = (
+                f"Document ID: {response['hits']['hits'][0]['_id']}\n"
+                f"Episode Info: {document['episode_info']}\n"
+                f"Video Path: {document['video_path']}\n"
+                f"Segment Text: {document.get('text', 'No text available')}\n"
+                f"Timestamp: {document.get('timestamp', 'No timestamp available')}"
+            )
+            await log_system_message(
+                logging.INFO, "Retrieved document:\n" + readable_output, logger,
+            )
         else:
             await log_system_message(logging.INFO, "No documents found.", logger)
 
     @staticmethod
-    async def index_transcriptions(base_path: str, es: AsyncElasticsearch, logger: logging.Logger) -> None:
+    async def index_transcriptions(
+        base_path: Path, es: AsyncElasticsearch, logger: logging.Logger,
+    ) -> None:
         actions = await ElasticSearchManager.__load_all_seasons_actions(base_path, logger)
 
         if actions:
-            await log_system_message(logging.INFO, f"Indexing {len(actions)} segments.", logger)
+            await log_system_message(
+                logging.INFO, f"Indexing {len(actions)} segments.", logger,
+            )
             await helpers.async_bulk(es, actions)
             await log_system_message(logging.INFO, "Data indexed successfully.", logger)
         else:
             await log_system_message(logging.INFO, "No data to index.", logger)
 
     @staticmethod
-    async def __load_all_seasons_actions(base_path: str, logger: logging.Logger) -> List[json]:
+    async def __load_all_seasons_actions(
+        base_path: Path, logger: logging.Logger,
+    ) -> List[dict]:
         actions = []
-        for season_dir in os.listdir(base_path):
-            season_path = os.path.join(base_path, season_dir)
-            if not os.path.isdir(season_path):
+        for season_path in base_path.iterdir():
+            if not season_path.is_dir():
                 continue
 
-            actions += await ElasticSearchManager.__load_season(logger, season_dir, season_path)
+            actions += await ElasticSearchManager.__load_season(logger, season_path)
         return actions
 
     @staticmethod
-    async def __load_season(logger: logging.Logger, season_dir: str, season_path: str) -> List[json]:
+    async def __load_season(
+        logger: logging.Logger, season_path: Path,
+    ) -> List[dict]:
```

Comment:
No i nie podobasz mi się, albo konkretny dict czegoś, albo json

Reply:
nwm co lepsze czy List[Dict[str, Dict[str, Dict[str, str]]]] czy json ale no niech będzie że ten dict jest bardziej ścisły

Reply:
nie no jak ładujesz jsona to json xD luźno, nie musisz wszystkiego rozpisywać